SUASSystem/SUASSystem/gcs.py

```diff
@@ -23,13 +24,15 @@ def gcs_process(sda_status, img_proc_status, interop_client_array, targets_to_su
     vehicle_state_data = manager.list()
     mission_information_data = manager.list()
     sda_avoid_coords = manager.list()
+    UAV_status = manager.Value('s', "AUTO")
     location_log = manager.list()
     vehicle = connect_to_vehicle()
     waypoints = download_waypoints(vehicle)
 
     if len(interop_client_array) != 0:
         mission_information_data.append(get_mission_json(interop_client_array[0].get_active_mission(), interop_client_array[0].get_obstacles()))
     else:
+        print('mission information is empty')
```

Comment:
Lets make this print statement a little more professional please. Something along the lines of:

```print("[Error] : The GCS process is unable to load mission data from the Interoperability server")```

Reply:
done